### ISSUE TITLE
No kbfs access on Windows

### ISSUE THREAD

**Issue by Scath-Enfys** [keybase/client#3765]
OS: Win10
log ID: 773811f39fb4a55ca7542b1c

kbfs simply won't mount anymore. Nor can I install keybase on anything other than an admin account, apparently.

**Comment by strib** [comment]
It looks like KBFS is mounting ok sometimes, but not every time:

```
782 2016-08-06T09:43:30.954483 - [ERRO kbfs mounter.go:55] 009 Failed to mount dokan filesystem (i=8): Dokan failed: -5
```

@zanderz @taruti any ideas?

**Comment by zanderz** [comment]
Status in your log upload says:
```{
   "Error": "Access is denied."
}```
This usually means the service is running under a different user account than the currently logged in user, which can happen when the installer is run with with admin permissions. I would recommend manually uninstalling both keybase and the dokan library, and running the installer again as regular user.

**Comment by taruti** [comment]
Per default we allow only the same user as is running kbfsdokan to access the drive.

**Comment by Scath-Enfys** [comment]
The main issue is that it would install under the admin user, not my current logged-in user. I tried to get around this by granting my normal user access to the install folder, but something is still amiss.

**Comment by taruti** [comment]
@Scath-Enfys kbfsdokan (the process serving K:) explicitely blocks other users than the one it is running under.

**Comment by damccull** [comment]
I'm having an issue where it won't mount the K drive too. The process appears to be running under my user, but it definitely prompts for admin access with UAC when dokan lib tries to install. The keybase installer works fine as a non-elevated user, but dokan will not install as a non-elevated user.

**Comment by zanderz** [comment]
Yes, sorry, I forgot to mention that UAC is expected and necessary only for the Dokan driver part. It is the user mode process that talks to it (kbfsdokan.exe) which must be installed and run as normal user, as part of Keybase.

**Comment by damccull** [comment]
Well, it still doesn't want to work...
![image](https://user-images.githubusercontent.com/401104/35633119-b3edc4e4-0665-11e8-9f60-9bf230d620d3.png)

I get this error when double clicking the K drive. This is after a fresh uninstall, reinstall, and even generating a new key for the computer. It shows up with the kbfs icon, but I get that error when trying to open it.

**Comment by zanderz** [comment]
@damccull Please do a keybase log send and post your ID

**Comment by damccull** [comment]
Ok, it's  57eff42ac37e760f70f2ac1c. Thanks for the help.

**Comment by zanderz** [comment]
@damccull Keybase considered your K: drive letter available on Jan 30 but mounting the filesystem failed. Logs show that a restart was still required after installing the Dokan part.

**Comment by damccull** [comment]
I'll try again, but I did restart it. I'll redo the install again.

**Comment by zanderz** [comment]
@rensa Please do `keybase log send` in a CMD window and let us know the log ID and we'll take a look.
